src/panels/calendar/ha-full-calendar.ts

```diff
@@ -442,6 +442,7 @@ export class HAFullCalendar extends LitElement {
 
         .fc-theme-standard .fc-scrollgrid {
           border: 1px solid var(--divider-color);
+          border-radius: 4px;
```

Comment:
```suggestion
          border-radius: var(--ha-card-border-radius, 12px);
```

Reply:
I had that originally, but that feels very round and I then went for the 4px that also the buttons above the calendar ("Today" and the modes) use. 

Should of course then use `var(--mdc-shape-small, 4px);`.

![image](https://user-images.githubusercontent.com/114137/205683709-81023601-c12b-4efb-9c93-08f9041263f0.png)

![image](https://user-images.githubusercontent.com/114137/205683812-e0de23a9-a714-4b09-9de5-e31e3dfe900a.png)

Reply:
@matthiasdebaat

Reply:
I would classify the calendar more like an input where we have 4px. E.g. in the scheduler form it looks otherwise a bit weird IMHO.

![image](https://user-images.githubusercontent.com/114137/205686636-17d88dc2-0fd7-4261-b370-3bd9cd23b261.png)

Reply:
> ![image](https://user-images.githubusercontent.com/114137/205683709-81023601-c12b-4efb-9c93-08f9041263f0.png)
> 
> ![image](https://user-images.githubusercontent.com/114137/205683812-e0de23a9-a714-4b09-9de5-e31e3dfe900a.png)

Let's use 4px. Combined with the other components it looks better.